new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCoherenceTests.java

```diff
@@ -0,0 +1,394 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getPodCreationTime;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+// Test to create a WebLogic domain with Coherence, build the Coherence proxy client program
+// which load and verify the cache.
+@DisplayName("Test to create a WebLogic domain with Coherence and verify the use of Coherence cache service")
+@IntegrationTest
+class ItCoherenceTests {
+
+  // constants for Coherence
+  private static final String PROXY_CLIENT_APP_NAME = "coherence-proxy-client";
+  private static final String PROXY_SERVER_APP_NAME = "coherence-proxy-server";
+  private static final String APP_LOC_ON_HOST = APP_DIR + "/" + PROXY_CLIENT_APP_NAME;
+  private static final String APP_LOC_IN_POD = "/u01/oracle/apps/" + PROXY_CLIENT_APP_NAME;
+  private static final String PROXY_CLIENT_SCRIPT = "buildRunProxyClient.sh";
+  private static final String OP_CACHE_LOAD = "load";
+  private static final String OP_CACHE_VALIDATE = "validate";
+  private static final String PROXY_PORT = "9000";
+
+  // constants for creating domain image using model in image
+  private static final String COHERENCE_MODEL_FILE = "coherence-wdt-config.yaml";
+  private static final String COHERENCE_MODEL_PROP = "coherence-wdt-config.properties";
+  private static final String COHERENCE_IMAGE_NAME = "coherence-image";
+
+  private static String domainUid = "coh-domain1";
+  private static String clusterName = "cluster-1";
+  private static String adminServerPodName = domainUid + "-admin-server";
+  private static String managedServerPrefix = domainUid + "-managed-server";
+  private static String containerName = "weblogic-server";
+  private static int replicaCount = 2;
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static Map<String, Object> secretNameMap;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void init(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
```

Comment:
The Namespace is already created by infra, here we are assigning a  namespace.  So better we should comment " Assigning a unique namespace for Operator  ( Domain ) "

Reply:
changed